src/app/ui/context_bar.cpp

```diff
@@ -1284,7 +1284,8 @@ class ContextBar::SymmetryField : public ButtonSet {
 
     DocumentPreferences& docPref = Preferences::instance().document(doc);
 
-    setSelectedItem((int)docPref.symmetry.mode());
+    at(0)->setSelected((int)docPref.symmetry.mode() == 1 || (int)docPref.symmetry.mode() == 3 ? true : false);
+    at(1)->setSelected((int)docPref.symmetry.mode() >= 2 ? true : false);
```

Comment:
There is no need to use the ternary operator, also I think that is better to use the SymmetryMode enum and not the numerical value, to improve code readability.
```suggestion
    app::gen::SymmetryMode symmetryMode = docPref.symmetry.mode()
    at(0)->setSelected(symmetryMode == app::gen::SymmetryMode::HORIZONTAL
                         || symmetryMode == app::gen::SymmetryMode::BOTH);
    at(1)->setSelected(symmetryMode == app::gen::SymmetryMode::VERTICAL ||
                         || symmetryMode == app::gen::SymmetryMode::BOTH);
```